mne/io/ctf_comp.py

```diff
@@ -67,13 +67,17 @@ def read_ctf_comp(fid, node, chs, verbose=None):
     chs : list
         The list of channels from info['chs'] to match with
         compensators that are read.
+    rename : dict | None
+        The channel renaming to use.
```

Comment:
as it's public API can you be more explicit? Something like 

    A dictionary mapping the old channel to a new channel name
    e.g. {'MLC11-606' : 'MLC11'}.

I took inspiration from rename_channels docstring  and used a CTF channel name

Reply:
I think it's mostly an accident that this is public -- I don't think users are ever meant to use it. I'd actually rather make it private (by deprecation if need be) rather than actually support people using it. It's misleading because they shouldn't ever need to use `ch_names_mapping` themselves, and if they do, they're not likely to do something good/correct with it. So I consider the vagueness here might actually be more of a feature than a bug...